### PR TITLE
feat(Node): add the Scheduler Node

### PR DESCRIPTION
**Pull request by agobrech**

### CODE REVIEW

**Comment by agobrech** [comment]
> @agobrech will this replace the Cron node?

Hi @luizeof, the scheduler is a combination of the Cron node and the Interval Node with some added features. Workflows that still have the Cron node or the Interval node will still work. You won't be able to find the old Cron node and the old Interval node in the node menu.

File: packages/nodes-base/nodes/Schedule/ScheduleTrigger.node.ts [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+					{
+						name: 'interval',
+						displayName: 'Trigger Interval',
+						values: [
+							{
+								displayName: 'Trigger Interval',
+								name: 'field',
+								type: 'options',
+								default: 'days',
+								options: [
```
Comment (krynble) [inline packages/nodes-base/nodes/Schedule/ScheduleTrigger.node.ts:65]: I know this list is ordered alphabetically but isn't it best if we order it from smallest to largest?

File: packages/nodes-base/nodes/Schedule/ScheduleTrigger.node.ts [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+								type: 'multiOptions',
+								displayOptions: {
+									show: {
+										field: ['weeks'],
+									},
+								},
+								typeOptions: {
+									maxValue: 7,
+								},
+								options: [
```
Comment (krynble) [inline packages/nodes-base/nodes/Schedule/ScheduleTrigger.node.ts:197]: Same here, start from `Sunday` to `Saturday` seems more intuitive

**Review by krynble — DISMISSED** [review]
I've added 2 minor comments about it, maybe we should change those. Other than this it's all good.

**Comment by netroy** [comment]
[We unified Cron expression generation a while ago](https://github.com/n8n-io/n8n/pull/3887). This change kinda reverts that by creating 2 different implementations again. 
can we perhaps we merge the two implementations to avoid inconsistent behaviors?

**Review by Joffcom — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.199.0`

**Pull request merged**